tests/python/test_sparse_deactivate.py

```diff
@@ -25,7 +25,7 @@ def func():
 
     @ti.kernel
     def deactivate():
-        ti.deactivate(ptr, 4)
+        ti.deactivate(ptr, 1)
```

Comment:
Hmm, I don't think this is relevant with this PR. That is, even before this change, changing this to `1` still passes the test.

IIUC, changing to consecutive indices requires us to change how `GlobalPtr` gets lowered to a chain of `GetCh` and `SNodeLookup` stmts. By having `print_ir=True`, you can see that there is a `bit_extract($2) bit_range=[4, 8)` stmt, where `$2` is the indices to be deactivated.  The `bit_range[4,8)` is what is actually causing the non-consecutive indices. @xumingkuan to confirm.

Reply:
So before this PR `ti.deactivate(ptr, 4)` would deactivate the first block. In fact, any index 0-15 would deactivate the first block. So to maintain the same exact behavior with with consecutive indices, I should have changed it to `ti.deactivate(ptr, 0)`. I just got lucky that `ti.deactivate(ptr, 1)` worked actually.  With this PR `ti.deactivate(ptr, 4)` defiently will not pass. Since it would deactivate the 5th block, which was never activated in the first place!

Reply:
will change to `ti.deactivate(ptr, 0)`

Reply:
Right, `bit_range=[4, 8)` will become `bit_range=[0, 4)` after this PR. `ti.deactivate(ptr, 0)` SGTM!